### ISSUE TITLE
gfs.gracefulify is not a function

### ISSUE THREAD

**Issue by leolorenzoluis** [tjunnone/npm-check-updates#380]
- [x] I have read the list of [known issues](https://github.com/tjunnone/npm-check-updates#known-issues) before filing this issue.
- [x] I have searched for [similiar issues](https://github.com/tjunnone/npm-check-updates/issues) before filing this issue.

* node version: 6.9.1
* npm version: 5.3.0
* npm-check-updates version: 2.12.1

I get the following error when running `ncu`

```
(node:26340) fs: re-evaluating native module sources is not supported. If you are using the graceful-fs module, please update it to a more recentversion.
/usr/local/lib/node_modules/npm-check-updates/node_modules/npm/lib/npm.js:18
  var fs = gfs.gracefulify(require('fs'))
               ^

TypeError: gfs.gracefulify is not a function
    at /usr/local/lib/node_modules/npm-check-updates/node_modules/npm/lib/npm.js:18:16
    at Object.<anonymous> (/usr/local/lib/node_modules/npm-check-updates/node_modules/npm/lib/npm.js:403:3)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/usr/local/lib/node_modules/npm-check-updates/node_modules/npmi/npmi.js:1:76)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
```

---------------------------

**Comment by raineorshine** [comment]
I'm sorry you're having this issue! I think this is due to a mismatch between the installed version of npm and the dependency in npm-check-updates. This is a known bug. We need to remove the npm dependency and always use this system version. This has been delayed because it is part of the v3 release and I have had to attend to other commitments. This is my top priority when I have a chance to work on this again!

**Comment by raineorshine** [comment]
Fixed in v3 branch via c0bd8b7.
